projectmanagement/src/main/java/life/qbic/projectmanagement/domain/project/experiment/Condition.java

```diff
@@ -121,47 +99,25 @@ public ConditionLabel label() {
    * @return true if <code>other</code> has the same levels defined; false otherwise
    */
   public boolean hasSameLevelsDefined(Condition other) {
```

Comment:
The function does now exactly the same as the overridden equals method. So redundant and can be removed.